app/src/main/java/com/example/trainingintive/dog_images_feature/presentation/AlbumFragment.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.trainingintive.dog_images_feature.presentation
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.trainingintive.R
+
+class AlbumFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
```

Comment:
Drop type and use `=`

Reply:
ahh, this pr is only scratch so I wasn't focused on such details ;)